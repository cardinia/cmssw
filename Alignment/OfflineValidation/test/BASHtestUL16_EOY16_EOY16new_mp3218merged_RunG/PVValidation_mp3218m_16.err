26-Sep-2019 13:05:58 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016G/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/72946285-3212-E911-991D-0CC47A4C8E70.root
26-Sep-2019 13:06:01 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016G/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/72946285-3212-E911-991D-0CC47A4C8E70.root
Begin processing the 1st record. Run 279071, Event 23734410, LumiSection 71 on stream 0 at 26-Sep-2019 13:06:16.500 CEST
Begin processing the 1001st record. Run 279071, Event 30161980, LumiSection 77 on stream 0 at 26-Sep-2019 13:08:50.692 CEST
Begin processing the 2001st record. Run 279071, Event 39529155, LumiSection 87 on stream 0 at 26-Sep-2019 13:10:56.493 CEST
Begin processing the 3001st record. Run 279071, Event 46829993, LumiSection 92 on stream 0 at 26-Sep-2019 13:13:32.099 CEST
Begin processing the 4001st record. Run 279071, Event 55097925, LumiSection 97 on stream 0 at 26-Sep-2019 13:16:04.516 CEST
Begin processing the 5001st record. Run 279071, Event 64111517, LumiSection 103 on stream 0 at 26-Sep-2019 13:18:40.052 CEST
Begin processing the 6001st record. Run 279071, Event 72912102, LumiSection 108 on stream 0 at 26-Sep-2019 13:21:13.911 CEST
Begin processing the 7001st record. Run 279071, Event 79937445, LumiSection 113 on stream 0 at 26-Sep-2019 13:23:44.796 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 13:23:57 CEST  Run: 279071 Event: 80096126
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 166
%MSG
Begin processing the 8001st record. Run 279071, Event 87382204, LumiSection 118 on stream 0 at 26-Sep-2019 13:26:17.455 CEST
Begin processing the 9001st record. Run 279071, Event 95057062, LumiSection 123 on stream 0 at 26-Sep-2019 13:28:52.202 CEST
Begin processing the 10001st record. Run 279071, Event 103065472, LumiSection 128 on stream 0 at 26-Sep-2019 13:31:22.088 CEST
Begin processing the 11001st record. Run 279071, Event 112402146, LumiSection 134 on stream 0 at 26-Sep-2019 13:33:55.596 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 13:34:54 CEST  Run: 279071 Event: 115029959
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 415
%MSG
Begin processing the 12001st record. Run 279071, Event 120423606, LumiSection 139 on stream 0 at 26-Sep-2019 13:36:30.315 CEST
Begin processing the 13001st record. Run 279071, Event 127883513, LumiSection 144 on stream 0 at 26-Sep-2019 13:39:01.726 CEST
Begin processing the 14001st record. Run 279071, Event 137142738, LumiSection 150 on stream 0 at 26-Sep-2019 13:41:33.118 CEST
Begin processing the 15001st record. Run 279071, Event 144304902, LumiSection 155 on stream 0 at 26-Sep-2019 13:43:58.525 CEST
Begin processing the 16001st record. Run 279071, Event 153848771, LumiSection 160 on stream 0 at 26-Sep-2019 13:46:24.666 CEST
Begin processing the 17001st record. Run 279071, Event 160189440, LumiSection 163 on stream 0 at 26-Sep-2019 13:48:48.403 CEST
Begin processing the 18001st record. Run 279071, Event 169292058, LumiSection 168 on stream 0 at 26-Sep-2019 13:51:12.024 CEST
Begin processing the 19001st record. Run 279071, Event 174878047, LumiSection 171 on stream 0 at 26-Sep-2019 13:53:37.481 CEST
Begin processing the 20001st record. Run 279071, Event 184412812, LumiSection 175 on stream 0 at 26-Sep-2019 13:56:03.388 CEST
Begin processing the 21001st record. Run 279071, Event 193059512, LumiSection 179 on stream 0 at 26-Sep-2019 13:58:27.715 CEST
Begin processing the 22001st record. Run 279071, Event 201320448, LumiSection 183 on stream 0 at 26-Sep-2019 14:00:57.986 CEST
Begin processing the 23001st record. Run 279071, Event 209608211, LumiSection 187 on stream 0 at 26-Sep-2019 14:03:21.267 CEST
Begin processing the 24001st record. Run 279071, Event 216416672, LumiSection 191 on stream 0 at 26-Sep-2019 14:05:46.789 CEST
Begin processing the 25001st record. Run 279071, Event 225307588, LumiSection 195 on stream 0 at 26-Sep-2019 14:08:09.043 CEST
Begin processing the 26001st record. Run 279071, Event 232131384, LumiSection 198 on stream 0 at 26-Sep-2019 14:10:33.294 CEST
Begin processing the 27001st record. Run 279071, Event 240296133, LumiSection 202 on stream 0 at 26-Sep-2019 14:13:00.141 CEST
Begin processing the 28001st record. Run 279071, Event 249073198, LumiSection 206 on stream 0 at 26-Sep-2019 14:15:20.254 CEST
Begin processing the 29001st record. Run 279071, Event 255580790, LumiSection 210 on stream 0 at 26-Sep-2019 14:17:46.438 CEST
Begin processing the 30001st record. Run 279071, Event 263911322, LumiSection 214 on stream 0 at 26-Sep-2019 14:20:12.620 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 14:21:55 CEST  Run: 279071 Event: 270411543
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 0
%MSG
26-Sep-2019 14:22:10 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016G/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/72946285-3212-E911-991D-0CC47A4C8E70.root
26-Sep-2019 14:22:10 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016G/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/00000/3455B62F-2012-E911-ABFE-0CC47A7452DA.root
26-Sep-2019 14:22:12 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016G/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/00000/3455B62F-2012-E911-ABFE-0CC47A7452DA.root
Begin processing the 31001st record. Run 279080, Event 165403617, LumiSection 191 on stream 0 at 26-Sep-2019 14:22:14.984 CEST
Begin processing the 32001st record. Run 279080, Event 169634253, LumiSection 195 on stream 0 at 26-Sep-2019 14:22:16.073 CEST
Begin processing the 33001st record. Run 279080, Event 174105588, LumiSection 200 on stream 0 at 26-Sep-2019 14:22:17.220 CEST
Begin processing the 34001st record. Run 279080, Event 178700098, LumiSection 208 on stream 0 at 26-Sep-2019 14:22:18.129 CEST
Begin processing the 35001st record. Run 279080, Event 184884912, LumiSection 212 on stream 0 at 26-Sep-2019 14:22:19.335 CEST
Begin processing the 36001st record. Run 279080, Event 188612686, LumiSection 215 on stream 0 at 26-Sep-2019 14:22:20.531 CEST
Begin processing the 37001st record. Run 279080, Event 193393227, LumiSection 219 on stream 0 at 26-Sep-2019 14:22:22.042 CEST
Begin processing the 38001st record. Run 279080, Event 197174755, LumiSection 222 on stream 0 at 26-Sep-2019 14:22:23.217 CEST
Begin processing the 39001st record. Run 279489, Event 3105049, LumiSection 2 on stream 0 at 26-Sep-2019 14:22:24.285 CEST
Begin processing the 40001st record. Run 279489, Event 13572946, LumiSection 7 on stream 0 at 26-Sep-2019 14:22:24.730 CEST
Begin processing the 41001st record. Run 279489, Event 20981011, LumiSection 11 on stream 0 at 26-Sep-2019 14:22:25.242 CEST
Begin processing the 42001st record. Run 279489, Event 28040527, LumiSection 15 on stream 0 at 26-Sep-2019 14:22:25.713 CEST
Begin processing the 43001st record. Run 279489, Event 37509065, LumiSection 19 on stream 0 at 26-Sep-2019 14:22:26.189 CEST
Begin processing the 44001st record. Run 279489, Event 47616365, LumiSection 24 on stream 0 at 26-Sep-2019 14:22:26.627 CEST
Begin processing the 45001st record. Run 279489, Event 55436534, LumiSection 28 on stream 0 at 26-Sep-2019 14:22:27.121 CEST
Begin processing the 46001st record. Run 279489, Event 64066155, LumiSection 32 on stream 0 at 26-Sep-2019 14:22:27.581 CEST
Begin processing the 47001st record. Run 279489, Event 74072206, LumiSection 37 on stream 0 at 26-Sep-2019 14:22:28.016 CEST
Begin processing the 48001st record. Run 279489, Event 81181545, LumiSection 41 on stream 0 at 26-Sep-2019 14:22:28.514 CEST
Begin processing the 49001st record. Run 279489, Event 89229142, LumiSection 45 on stream 0 at 26-Sep-2019 14:22:29.095 CEST
26-Sep-2019 14:22:29 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016G/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/00000/3455B62F-2012-E911-ABFE-0CC47A7452DA.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 OutOfBounds          -w TrackRefitter:Fi                       3        3
    2 fileAction           -s file_close                             2        2
    3 fileAction           -s file_open                              4        4

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 OutOfBounds          279071/80096126  279071/115029959 279071/270411543
    2 fileAction           PostProcessEvent PostGlobalEndRun 
    3 fileAction           pre-events       pre-events       PostProcessEvent

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
Warning                 3                   3
System                  6                   6

dropped waiting message count 0
[9.338MB/9.338MB][100%][==================================================][9.338MB/s]  [9.338MB/9.338MB][100%][==================================================][9.338MB/s]  
