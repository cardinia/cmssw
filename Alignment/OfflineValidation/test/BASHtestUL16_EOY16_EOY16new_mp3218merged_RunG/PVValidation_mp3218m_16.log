000 (3189587.016.000) 09/26 11:25:05 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3189587.016.000) 09/26 13:05:49 Started transferring input files
	Transferring to host: <188.184.138.44:9618?addrs=188.184.138.44-9618+[2001-1458-301-45--100-2c]-9618&noUDP&sock=32612_daa7_130357>
...
040 (3189587.016.000) 09/26 13:05:49 Finished transferring input files
...
001 (3189587.016.000) 09/26 13:05:49 Job executing on host: <188.184.138.44:9618?addrs=188.184.138.44-9618+[2001-1458-301-45--100-2c]-9618&noUDP&sock=5608_3334_3>
...
006 (3189587.016.000) 09/26 13:05:58 Image size of job updated: 243436
	124  -  MemoryUsage of job (MB)
	126096  -  ResidentSetSize of job (KB)
...
006 (3189587.016.000) 09/26 13:10:59 Image size of job updated: 663336
	541  -  MemoryUsage of job (MB)
	553724  -  ResidentSetSize of job (KB)
...
006 (3189587.016.000) 09/26 14:11:07 Image size of job updated: 663468
	541  -  MemoryUsage of job (MB)
	553856  -  ResidentSetSize of job (KB)
...
040 (3189587.016.000) 09/26 14:22:46 Started transferring output files
...
040 (3189587.016.000) 09/26 14:22:46 Finished transferring output files
...
005 (3189587.016.000) 09/26 14:22:46 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 01:15:56, Sys 0 00:00:08  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 01:15:56, Sys 0 00:00:08  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	10396  -  Run Bytes Sent By Job
	1015  -  Run Bytes Received By Job
	10396  -  Total Bytes Sent By Job
	1015  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :     9611        1   2204890 
	   Memory (MB)          :      541     2000      2000 
...
