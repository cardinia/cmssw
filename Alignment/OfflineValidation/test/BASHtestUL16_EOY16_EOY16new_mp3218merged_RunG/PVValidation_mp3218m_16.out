Job started at  Thu Sep 26 13:05:50 CEST 2019
batch dir: /pool/condor/dir_20915 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_279071_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 9.4M -rw-r--r--. 1 tkello zh 9.4M Sep 26 14:22 PVValidation_mp3218m_279071.root -rw-r--r--. 1 tkello zh 21K Sep 26 13:05 PVValidation_mp3218m_279071_cfg.py -rw-r--r--. 1 tkello zh 9.0K Sep 26 14:22 _condor_stderr -rw-r--r--. 1 tkello zh 522 Sep 26 13:05 _condor_stdout -rwxr-xr-x. 1 tkello zh 1015 Sep 26 13:05 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 26 13:05 tkello.cc drwx------. 3 tkello zh 4.0K Sep 26 13:05 tmp drwx------. 3 tkello zh 4.0K Sep 26 13:05 var
Job ended at  Thu Sep 26 14:22:46 CEST 2019
